ecosystem/typescript/sdk/src/tests/unit/account_module.test.ts

```diff
@@ -0,0 +1,68 @@
+import { AptosClient } from "../../providers";
+import { longTestTimeout, NODE_URL } from "./test_helper.test";
+import * as aptosClient from "../../client";
+import { MoveFunctionVisibility } from "../../generated";
+
+jest.mock("../../client", () => {
+  return {
+    __esModule: true,
```

Comment:
is this needed?

Reply:
Yes. it will complain if I don't add it

e.g.
```
TypeError: Cannot redefine property: get
        at Function.defineProperty (<anonymous>)

      42 |       ];
      43 |
    > 44 |       const getSpy = jest.spyOn(aptosClient, "get").mockResolvedValue({
```

I wanted to mock http calls with [nock](https://github.com/nock/nock), but didn't want to introduce a new dependency for this simple change.